spec/lib/gettext_setup_spec.rb

```diff
@@ -50,8 +52,10 @@
       expect(GettextSetup.default_locale).to eq('en')
       expect(GettextSetup.candidate_locales).to include('en')
       GettextSetup.clear
+      before_locale = ENV['LANG']
       ENV['LANG'] = 'de_DE'
       expect(GettextSetup.candidate_locales).to eq('de_DE,de,en')
+      ENV['LANG'] = before_locale
```

Comment:
That might be a good idea. I can add it to both sets, to make things more reliable.